lib/u3d/unity_versions.rb

```diff
@@ -113,7 +114,9 @@ def list_available
             if response.is_a? Net::HTTPSuccess
               capt = response.body.match(LINUX_DOWNLOAD_RECENT_FILE)
               if capt && capt[1] && capt[2]
-                versions[capt[2].delete('x')] = capt[1]
+                ver = capt[2].delete('x')
```

Comment:
I would put some warnings here to see if the `ver` does not satisfy our sanitized version names.